types/express-http-proxy/index.d.ts

```diff
@@ -52,6 +52,7 @@ declare namespace proxy {
         reqAsBuffer?: boolean;
         reqBodyEncoding?: string | null;
         timeout?: number;
+        proxyReqBodyDecorator?: (bodyContent: any, srcReq: Request) => any;
```

Comment:
Can you please add typings for the promise form?

Reply:
Additionally, for style, can you please group all the callbacks together?

Reply:
I think `any` includes the `Promise<any>` as well